app/src/pages/project/StreamToggle.tsx

```diff
@@ -13,8 +14,24 @@ import { StreamToggle_data$key } from "./__generated__/StreamToggle_data.graphql
  */
 const REFRESH_INTERVAL_MS = 2000;
 
+/**
+ * Routes where streaming is enabled
+ */
+const STREAMING_ENABLED_ROUTES = ["/spans", "/traces", "/sessions"];
```

Comment:
`STREAMING_ENABLED_ROUTE_TAILS`? This took me a minute to realize that only if the path ends with one of these streaming will be enabled.

Reply:
makes sense